eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -684,6 +689,13 @@ debugPlutus version db =
                       (getEvaluationContext costModel)
                       (transExUnits exUnits)
                       scripts
+                  evalV3 =
```

Comment:
Perhaps there is a missing addition of
```haskell
, plutusDebugLangDecoder (Proxy @'PlutusV3)
```
as part of the list argument to `F.asum` in `plutusDebugDecoder`, a few lines above this in the same function. I am not certain, but it looked like it. I may be wrong.

Reply:
yes indeed, thank you!! I wish I knew of a way to get the type checker to catch this for us... :thinking:

Reply:
Me too! :+1: